xdcs-frontend/src/app/app-routing.module.ts

```diff
@@ -16,9 +18,10 @@ export const routes: Routes = [
       { path: '', redirectTo: 'home', pathMatch: 'full' },
       { path: 'home', component: HomeComponent },
       { path: 'task-definitions', component: TaskDefinitionsComponent },
-      { path: 'task-definitions/:id', component: TaskDefinitionComponent },
+      {path: 'task-definitions/:definitionId', component: TaskDefinitionComponent},
```

Comment:
w PPXDCS-104 dodałem na to regułę tslinta

Reply:
done